exes/mqtt-broadcaster/inc/config.hpp

```diff
@@ -1,28 +1,50 @@
 #pragma once
 
 #include <tfc/confman.hpp>
-#include <tfc/confman/observable.hpp>
+
+struct signal_defintion {
+  std::string name{};
+  tfc::ipc::details::type_e type{};
```

Comment:
```suggestion
  tfc::ipc::details::type_e type{tfc::ipc::details::type_e::unknown};
```
pass in the default here as unknown